schematic/manifest/commands.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import click
+from schematic.manifest.generator import ManifestGenerator
+from schematic import CONFIG
+
+CONTEXT_SETTINGS = dict(help_option_names=['--help', '-h'])  # help options
+
+# invoke_without_command=True -> forces the application not to show aids before losing them with a --h
+@click.group(context_settings=CONTEXT_SETTINGS, invoke_without_command=True)
+def manifest(): # use as `schematic manifest ...`
+    """
+    Sub-commands with Manifest Generation utilities/methods.
+    """
+    pass
+
+
+# prototype based on getModelManifest() and get_manifest()
+# use as `schematic config get positional_args --optional_args`
+@manifest.command('get', short_help='Prepares the manifest URL based on provided schema.')
```

Comment:
We might want to better distinguish between mandatory or optional arguments by switching some arguments to being positional.

Reply:
We agreed that we wouldn't have any positional arguments while defining our CLIs. All arguments will be optional arguments. To enforce certain arguments we would just create "required, optional" arguments.